packages/config/config/devices/0x018c/pm-b400zw-n.json

```
@@ -0,0 +1,73 @@
+{
+	"manufacturer": "Dawon DNS",
+	"manufacturerId": "0x018c",
+	"label": "PM-B400ZW-N",
+	"description": "Power Manager",
+	"devices": [
+		{
+			"productType": "0x0042",
+			"productId": "0x0006",
+			"zwaveAllianceId": 1870
+		},
+		{
+			"productType": "0x0042",
+			"productId": "0x0005"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Group 1",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Standy Power Setting Value",
+			"description": "If 300, standby power is 3.00w and standby power is enabled. default : 0, disabled",
+			"valueSize": 4,
+			"unit": "0.01 W",
+			"minValue": 0,
+			"maxValue": 60000,
+			"defaultValue": 0
+		},
+		{
+			"#": "2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Periodic kWh Reporting",
+			"defaultValue": 1
+		},
+		{
+			"#": "3",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Accumulation of kWh Values Stop (0) /Start (1)",
+			"defaultValue": 1
+		},
+		{
+			"#": "4",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Connected Device in Use (1) /Not in Use (0)"
+		},
+		{
+			"#": "5",
+			"label": "Periodic kWh Reporting Time",
```

```suggestion
			"label": "Energy (kWh) Report Interval",
```
for consistency with other devices.

---

Ok